packages/kmod-6.1-nvidia/kmod-6.1-nvidia.spec

```
@@ -111,6 +144,12 @@ rm nvidia{,-modeset,-peermem}.o
 # don't include any linked module in the base image
 rm nvidia{,-modeset,-peermem,-drm}.ko
 
+# End proprietary driver build
+popd
+
+# Grab the list of supported devices
+pushd NVIDIA-Linux-%{_cross_arch}-%{tesla_ver}/supported-gpus
+cat supported-gpus.json | jq -r '.chips[] | select(.features[] | contains("kernelopen"))' | jq -s '{"open-gpu": .}' > open-gpu-supported-devices.json
```

nit: useless use of `cat`
```suggestion
jq -r '.chips[] | select(.features[] | contains("kernelopen"))' supported-gpus.json | jq -s '{"open-gpu": .}' > open-gpu-supported-devices.json
```

Wouldn't hurt to add a check to ensure that you can pull out a leaf value you expect to find, in case the structure changes slightly.

---

Good call!